.ci/Jenkinsfile

```diff
@@ -344,12 +344,11 @@ def generateFunctionalTestStep(Map args = [:]){
                 if(isInstalled(tool: 'docker', flag: '--version')) {
                     dockerLogin(secret: "${DOCKER_ELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
                 }
+                filebeat(image: 'docker.elastic.co/beats/filebeat:7.13.0-SNAPSHOT', output: "docker_logs_${platform}_${goArch}_${suite}_${name}.log", workdir: "${env.WORKSPACE}") {
```

Comment:
Are you sure you want to pin to a snapshot? Maybe it's better to pin to a released version?

Reply:
Good point. I remember we pinned to the snapshot when the original step was using an older version. I think it's safe now to use whatever in the default value. Let me remove that version once the CI passes.

Reply:
@cachedout the last commit is just code indentation. Please review it commits without taking a look at the last one to see the real changes in here